src/locales/helpers.tsx

```diff
@@ -0,0 +1,26 @@
+import i18next from 'i18next';
```

Comment:
I would assume to find locales in a `locales` folder. Can we rename it to `utils`?

Reply:
There is also a `util.ts`file in the src directory. Should I move it within this new `utils` folder too? or do I just rename this file to `localesUtils.tsx` and put it directly in the src directory like util.ts?

Reply:
Yes, `util.ts` seems a good place for it.

Reply:
Actually, with the new structure it makes sense where it is.